src/controllers/__tests__/auth.test.js

```diff
@@ -26,8 +25,8 @@ const wrongUser = {
 
 describe('POST /auth', () => {
   afterAll(async () => {
-    await Volunteer.deleteMany();
-    await Ngo.deleteMany();
+    await User.deleteMany();
+    await User.deleteMany();
```

Comment:
Hi @abdulhafiz96 do we need to use User.deleteMany() twice? Or maybe I misread it.